serving/src/test/java/feast/serving/service/OnlineServingServiceTest.java

```diff
@@ -181,7 +103,7 @@ public void shouldReturnKeysWithoutVersionIfNotProvided() {
                         Field.newBuilder().setName("entity2").setValue(strValue("b")).build(),
                         Field.newBuilder().setName("feature1").setValue(intValue(2)).build(),
                         Field.newBuilder().setName("feature2").setValue(intValue(2)).build()))
-                .setFeatureSet("featureSet")
+                .setFeatureSet("project/featureSet")
```

Comment:
Would the default project result in a feature set of `featureSet` or `default/featureSet`? Do we have a test for the default case?

Reply:
> Would the default project result in a feature set of featureSet or default/featureSet? 

Pending  @zhilingc 's PR  #701 

> Do we have a test for the default case?

We not need to test for the default case here as `OnlineServingService` never interacts with the `feature_set` field of the feature row. (ie The tests would pass even if `feature_set` is unset in the `FeatureRow`)